ui/src/app/applications/components/application-resources-diff/application-resources-diff.scss

```diff
@@ -0,0 +1,14 @@
+.application-component-diff {
```

Comment:
please rename `application-component-diff` to `application-resources-diff`

Reply:
I've hidden unmodified files.